src/main/java/com/homihq/db2rest/auth/service/UnkeyDevService.java

```diff
@@ -0,0 +1,49 @@
+package com.homihq.db2rest.auth.service;
+
+import com.homihq.db2rest.auth.to.VerifyKeyRequest;
+import com.homihq.db2rest.auth.to.VerifyKeyResponse;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class UnkeyDevService {
+
+    private final RestTemplateBuilder restTemplateBuilder;
+
+
+    String UNKEY_DEV_DOMAIN = "https://api.unkey.dev/v1";
```

Comment:
This should be an environment variable. This is because unkey can be deployed without the cloud version.

Reply:
implemented